<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:10pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(14 , 150 , 254);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(44 , 137 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(14 , 135 , 228);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sign">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:10pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(14 , 150 , 254);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(44 , 137 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(14 , 135 , 228);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 127);</string>
   </property>
   <property name="text">
    <string>username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwd_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 127);</string>
   </property>
   <property name="text">
    <string>password：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_username">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 1px solid gray; /* 边框 */
	border-radius: 10px;     /* 边框圆角 */
	padding: 0 8px;         /* 内容与边框距离 */
	background: white;      /* 背景颜色 */
	selection-background-color: darkgray; /*所选内容的背景颜色*/
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passwd">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 1px solid gray; /* 边框 */
	border-radius: 10px;     /* 边框圆角 */
	padding: 0 8px;         /* 内容与边框距离 */
	background: white;      /* 背景颜色 */
	selection-background-color: darkgray; /*所选内容的背景颜色*/
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 127);

</string>
   </property>
   <property name="text">
    <string>uid:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uid">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 1px solid gray; /* 边框 */
	border-radius: 10px;     /* 边框圆角 */
	padding: 0 8px;         /* 内容与边框距离 */
	background: white;      /* 背景颜色 */
	selection-background-color: darkgray; /*所选内容的背景颜色*/
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
